### DISCUSSION TITLE
PowerShell error: Multiple files needs merge (can't update Firefox UI Fix)

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#937] [Q&A]
I can't update Firefox UI Fix using PowerShell since it pops out multiple errors and some files can't be merged.

I don't want to create a new "chrome" folder in my profile since I would have to reinstall Firefox Scripts & CustomCSSforFx too.

UPDATE: I manually replaced the css, icons & src folders, but I still need help to fix PowerShell since I'm regularly using it for Firefox UI Fix updates and it's a little bit annoying to every time download the source code and replace all files in folders in chrome folder.

Here is a screenshot of my PowerShell window:
![PowerShell error files needs merge](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/4ba8ec9d-1a5d-44d8-a7d1-4ee0593cc360)

**Comment by Aleksandra0056** [comment]
I no longer have this problem, because I re-copied all files from the old `chrome` folder to the new one.

But I think this discussion should be kept open because more people could experience similar problems.

**Reply by black7375** [reply]
Obviously, there may be conflicts if you're doing things like modifying icons.

**Reply by Aleksandra0056** [reply]
Yes, I modified some icons of extensions or menu items using userChrome.css in past, but I thought, that it shouldn't cause any problems.
